src/KDFoundation/platform/abstract_platform_timer.h

```diff
@@ -15,10 +15,9 @@
 
 namespace KDFoundation {
 
-class KDFOUNDATION_API AbstractPlatformTimer
-{
-public:
-    virtual ~AbstractPlatformTimer() { }
+class KDFOUNDATION_API AbstractPlatformTimer{
```

Comment:
This formatting is not right. Seems like `clang-format` regression:
https://github.com/llvm/llvm-project/issues/112526